gptme/tools/base.py

```diff
@@ -90,12 +142,39 @@ def __eq__(self, other):
             return False
         return self.name == other.name
 
+    def is_executable(self):
+        return bool(self.execute)
+
+    def get_instructions(self, tool_format: ToolFormat):
+        if tool_format in self.instructions_format:
+            return "\n".join([self.instructions, self.instructions_format[tool_format]])
```

Comment:
Assuming both are stripped strings:

```suggestion
            return "\n\n".join([self.instructions, self.instructions_format[tool_format]])
```

Reply:
Fixed